API.md

```diff
@@ -1004,7 +1020,7 @@ item: {
 }
 ```
 
-For instance, if you have specified the data like this 
+For instance, if you have specified the data like this
```

Comment:
I think atom cleans up the spaces at the end of the lines.

As for the doc checking, we'll see